frontend/src/components/AccountSentTable.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <div>
```

Comment:
That works for me

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/546/commits/d35dc09f392fd684e1b325aa294c4d86723b50e0

Reply:
Sorry for the churn here—this looks much better, but instead of just the text/clickable icon can we make the whole thing the same style of "button" we use for things like the "Copy payment link" / "Send history" button here:

<img width="620" alt="Screen Shot 2023-06-30 at 5 27 01 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/b4d12977-0b69-4e72-9caa-fd8d4c7255c0">

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/546/commits/e0da8a94bda6b4be63a44fb4a5b101fcbaa31a5d styled as button